package-lock.json

```diff
@@ -2385,7 +2385,7 @@
     "request": {
```

Comment:
We might need to update the `request` to update the `hoek`

Reply:
└─┬ vscode@1.1.14
  ├─┬ gulp-remote-src@0.4.3
  │ └─┬ request@2.79.0
  │   └─┬ hawk@3.1.3
  │     ├─┬ boom@2.10.1
  │     │ └── hoek@2.16.3  deduped
  │     ├── hoek@2.16.3 
  │     └─┬ sntp@1.0.9
  │       └── hoek@2.16.3  deduped
  └─┬ request@2.85.0
    └─┬ hawk@6.0.2
      ├─┬ boom@4.3.1
      │ └── hoek@4.2.1  deduped
      ├─┬ cryptiles@3.1.2
      │ └─┬ boom@5.2.0
      │   └── hoek@4.2.1  deduped
      ├── hoek@4.2.1 
      └─┬ sntp@2.1.0
        └── hoek@4.2.1  deduped



wait for them to fix